Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityDeviceManager.cs

```diff
@@ -130,7 +133,7 @@ private WindowsMixedRealityController GetOrAddController(InteractionSourceState
         /// <param name="interactionSourceState">Source State provided by the SDK to remove</param>
         private void RemoveController(InteractionSourceState interactionSourceState)
         {
-            var controller = GetOrAddController(interactionSourceState, false);
```

Comment:
Why change/ remove this.  Had to add this as it caused issues on startup, unless you've resolve them?

Reply:
I resolved them.

Reply:
Besides this parameter was not being used.

Reply:
It was being used here:

![image](https://user-images.githubusercontent.com/3580640/43663204-6a8ca894-971d-11e8-8b86-d28c5af29dcc.png)

Reply:
Yup, that was the second Update we were talking about that gets called.

Reply:
Getting the controller shouldn't update it.

It should be a discrete step

Reply:
Agreed.